src/components/Media/VideoMedia/index.tsx

```diff
@@ -23,20 +25,23 @@ export const VideoMedia = (props: MediaProps) => {
   }, [])
 
   if (resource && typeof resource === 'object') {
-    const { filename } = resource
+    const { url } = resource
+    const cacheTag = resource.updatedAt
+
+    const src = getMediaURL(url, cacheTag, getHostnameFromTenant(tenant))
 
     return (
       <video
         autoPlay
         className={cn(videoClassName)}
-        controls={false}
+        controls={showVideoControls || true}
```

Comment:
I think we should use nullish coalescing here because if `showVideoControls` is passed as `false` it would actually evaluate to `true` with the use of `||` here. 

```suggestion
        controls={showVideoControls ?? true}
```